File: Problems/Leetcode/3001-4000/3101-3200/3192.cpp
#include <bits/stdc++.h>

using namespace std;

static constexpr int MOD = 1e9 + 7;

class Solution {
public:
    int minOperations(vector<int>& nums) {
        vector<int> res(nums);
        res.erase(unique(res.begin(), res.end()), res.end());
        return count(res.begin(), res.end(), 0) * 2 - (res.back() == 0);
    }
};

int main() {
    Solution sol;

    return 0;
}

File: Problems/Leetcode/3001-4000/3101-3200/3192.py
from itertools import pairwise
from typing import List


class Solution:
    def minOperations(self, nums: List[int]) -> int:
        k = 0
        for x in nums:
            if x == k % 2:
                k += 1
        return k

        # return (nums[0] ^ 1) + sum(x ^ y for x, y in pairwise(nums))
